jobs/harbor/templates/bin/pre-start.erb

```diff
@@ -166,5 +166,13 @@ fi
 #Save installed harbor version to file
 echo $HARBOR_FULL_VERSION > $HARBOR_VERSION_FILE
 
+<%- if p("enable_externalssl") -%>
+################# copy nginx.conf file
+/bin/cp ${HARBOR_JOB_DIR}/config/nginx.conf ${HARBOR_PACKAGE_DIR}/common/config/nginx/nginx.conf
```

Comment:
harbor package itself contains https://github.com/vmware/harbor/blob/master/make/common/templates/nginx/nginx.https.conf. What's the difference of your nginx.conf with the nginx.https.conf?